### DISCUSSION TITLE
Reading Logs from AWS CloudWatch

### DISCUSSION THREAD

**Discussion by dev-mo-ali** [opcodesio/log-viewer#330] [Ideas]
Hi
hope if we can make it possible to read the logs from AWS CloudWatch
The scenario is: multiple Laravel EC2 that logs into AWS CloudWatch as centralized logs, will be great if can read that logs using this package as well 

thanks

**Comment by arukompas** [comment]
Hey @dev-mo-ali 

Currently Log Viewer can only read logs from the local file system. And so far there are no plans to support remote log files or APIs.

But if you have the time and want to tinker around, consider opening a PR with a new LogReader class such as CloudwatchLogReader.
